components/rmrk/Gallery/CollectionItem.vue

```diff
@@ -293,22 +294,40 @@ export default class CollectionItem extends mixins(
     this.checkId()
     this.checkActiveTab()
     this.checkIfEmptyListed()
-    this.$apollo.addSmartQuery('collection', {
+    this.fetchPageData(this.startPage)
+  }
+
+  public async fetchPageData(page: number, loadDirection = 'down') {
+    if (this.isFetchingData) return false
+    this.isFetchingData = true
+    const result = await this.$apollo.query({
       query: collectionById,
       client: this.urlPrefix,
-      loadingKey: 'queryLoading',
-      manual: true,
-      variables: () => {
-        return {
-          id: this.id,
-          orderBy: this.searchQuery.sortBy,
-          search: this.buildSearchParam(),
-          first: this.first,
-          offset: this.offset,
-        }
+      variables: {
```

Comment:
Because `$apollo.query` need `variables` to be an object.

Reply:
🤔 
https://apollo.vuejs.org/guide/apollo/queries.html#reactive-parameters

Reply:
https://apollo.vuejs.org/guide/apollo/queries.html#option-function
<img width="795" alt="image" src="https://user-images.githubusercontent.com/31397967/165754018-0bd9011c-f9d7-478f-9da1-12f508831e83.png">


https://apollo.vuejs.org/guide/apollo/queries.html#reactive-query-definition

<img width="783" alt="image" src="https://user-images.githubusercontent.com/31397967/165754160-5742ce0c-a162-45e8-b020-e082d70d81e5.png">

Reply:
Both `function` and `object` could work.

However, the query does not need to be 'reactive' in this case.
So I use the object to be the param.

Reply:
I changed my mind after doing some tries, because using `function` as `variables` could not work.

`$apollo.query` only accept 'object' in this case.

<img width="732" alt="image" src="https://user-images.githubusercontent.com/31397967/166139913-f744fbe6-71d7-404b-b78d-81fd295d17e4.png">